### ISSUE TITLE
Support for string dates

### ISSUE THREAD

**Issue by nviitala** [nathanreyes/v-calendar#345]
Hi,

It seems that the v-date-picker's initial value can be set only from Date object.

So when I fetch data from API as JSON I need to convert the date string first to Date object. It would be more convenient if the system would support strings too, just like you can define the input like `input: ['DD.MM.YYYY', 'YYYY-MM-DD', 'YYYY/MM/DD']`.

Kind regards,
Niko

**Comment by brnmonteiro** [comment]
nice feature

**Comment by vciloglu** [comment]
Yes !. that would be great. it takes me time to set this up in my incoming data.
I have that same problem

**Comment by vasilestefirta** [comment]
+1 on this feature. I needed to use a string value in Laravel blade template and I ended up doing it this way:
```html
<v-date-picker
    :value="new Date('{{ $dateString }}')"
    :min-date="new Date()"
    :masks="{ input: 'MMMM D, YYYY' }"
    is-dark
/>
```

Where `$dateString` can be something like `06/18/2020`. It would be nice to just pass a string date to the `value` prop directly.

**Comment by rasensio** [comment]
+1 on this feature. Doing a conversion of the data is a total bummer and makes no sense. Should be hidden from the implementation. 
Anyways, thanks for doing this ,v-calendar is by far the best Vue Calendar out there.

**Comment by renduh** [comment]
**+1 for this feature.**
Brilliant Vue Calendar but it would be even better if you could pass a string as the date value.

**Comment by rifton007** [comment]
I have the same problem.

May API return this date for example : "2020-09-30T22:00:00.000Z"
And Datepicker does'nt retrive this date.
```
  <vc-date-picker
                                    v-model="birthday"
                                    :value="new Date(birthday)"
                                    :max-date="new Date()"
                                    :popover="{ visibility: 'click' }"
                                    :input-props='{
                                        class: "form-control rounded-0",
                                    }'
                                />
```

**Comment by garbinmarcelo** [comment]
Hello, 

I am facing the same problem. Is it currently not possible to pass the date value as a string?

I'm using [Vuesax](https://lusaxweb.github.io/vuesax/components/input.html#size) input where it accepts only string and number as value props

thank's
